src/shims/ApiCompatBaseline.uap.netstandard20.txt

```diff
@@ -1 +1,8 @@
-Total Issues: 0
+Compat issues with assembly mscorlib:
```

Comment:
Should there be an issue around this? How come do we still have missing netstandard2.0 api?